src/CommunityToolkit/Xamarin.CommunityToolkit/Views/TabView/TabView.shared.cs

```diff
@@ -745,12 +745,10 @@ bool CanUpdateSelectedIndex(int selectedIndex)
 			return true;
 		}
 
-		void UpdateSelectedIndex(int position, bool hasCurrentItem = false)
+		void UpdateSelectedIndex(int oldPosition, int position, bool hasCurrentItem = false)
```

Comment:
Is it simpler perhaps, to validate the old and new index before invoking UpdateSelectedIndex?. Could you add an example where validate the fix?